README.md

```diff
@@ -209,12 +209,31 @@ kustomize build common/istio-1-9-0/kubeflow-istio-resources/base | kubectl apply
 
 #### Kubeflow Pipelines
 
-Install the Kubeflow Pipelines official Kubeflow component:
+Install the Multi-User Kubeflow Pipelines official Kubeflow component:
 
 ```sh
 kustomize build apps/pipeline/upstream/env/platform-agnostic-multi-user | kubectl apply -f -
```

Comment:
Maybe it is a good idea to mention `apps/pipeline/upstream/env/platform-agnostic-multi-user-pns` for those that aren't using Docker as their container runtime.